src/librbd/io/ObjectRequest.cc

```diff
@@ -20,7 +20,7 @@
 #include "librbd/io/ImageRequest.h"
 #include "librbd/io/ReadResult.h"
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
Nit: just drop the include since it appears not to be used anymore

Reply:
Fixed